plugin-gradle/src/main/java/com/diffplug/gradle/spotless/GradleProvisioner.java

```diff
@@ -121,7 +122,14 @@ private static Provisioner forConfigurationContainer(Project project, Configurat
 				config.setCanBeConsumed(false);
 				config.setVisible(false);
 				config.attributes(attr -> {
-					attr.attribute(Bundling.BUNDLING_ATTRIBUTE, project.getObjects().named(Bundling.class, Bundling.EXTERNAL));
+					final String type;
+					// See https://github.com/diffplug/spotless/pull/1808#discussion_r1321682984.
+					if (mavenCoords.stream().anyMatch(it -> it.startsWith(KtLintStep.MAVEN_COORDINATE_1_DOT))) {
+						type = Bundling.SHADOWED;
```

Comment:
This will resolve [ktlint-cli-1.0.0-all.jar](https://repo.maven.apache.org/maven2/com/pinterest/ktlint/ktlint-cli/1.0.0/ktlint-cli-1.0.0-all.jar) instead of [ktlint-cli-1.0.0.jar](https://repo.maven.apache.org/maven2/com/pinterest/ktlint/ktlint-cli/1.0.0/ktlint-cli-1.0.0.jar) for now.